index.d.ts

```diff
@@ -0,0 +1,47 @@
+declare namespace bitbar {
+	// Options for the original BitBar program, taken from here:
+	// https://github.com/matryer/bitbar#plugin-api
+	interface BitbarOptions {
+		// The only required property.
+		text: string;
+
+		href?: string;
+		color?: string;
+		font?: string;
+		size?: number;
+		bash?: string;
+
+		// There are up to 5 params hardcoded into the program.
+		param1?: string;
+		param2?: string;
+		param3?: string;
+		param4?: string;
+		param5?: string;
+
+		terminal?: boolean;
+		refresh?: boolean;
+		dropdown?: boolean;
+		length?: number;
+		trim?: boolean;
+		alternate?: boolean;
+		templateImage?: string;
+		image?: string;
+		emojize?: boolean;
+		ansi?: boolean;
```

Comment:
Can you add doc comments for each property? I know it's some work, but it will benefit a lot of people to have direct comments when using the properties.